src/MarkdownEditor.js

```diff
@@ -78,15 +91,15 @@ var MarkdownEditor = React.createClass({
     };
 
     if(this.props.styles.styleMarkdownEditorContainer) {
-      styleMarkdownEditorContainer = this.props.styles.styleMarkdownEditorContainer;
+      Object.assign(styleMarkdownEditorContainer, this.props.styles.styleMarkdownEditorContainer);
     }
 
     return (
-      <div
+      <div id='container'
```

Comment:
whoops... this was a debugging item for my linked project to make sure I was working with the element I wanted. I'll remove